src/main/plugins/org.dita.base/init/startup/topics/welcome.dita

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE topic PUBLIC "-//OASIS//DTD DITA Topic//EN" "topic.dtd">
+<topic id="welcome">
+  <title>My first topic</title>
+  <body>
+    <p>Welcome to your first <ph keyref="product"/>.</p>
+    <p>You can add images:</p>
+    <image href="../images/welcome.png" id="image_j3n_cc1_bpb" placement="break"/>
+    <p>You can also use reusable content:</p>
+    <note conref="../reusables/reusable_components.dita#reusable_notes/reusable_note" id="note_rtg_hc1_bpb"/>
```

Comment:
Unreferenced generated IDs here, which we typically [avoid](https://github.com/dita-ot/docs/wiki/coding-guidelines#avoid-unnecessary-markup).

Reply:
Again same thing, this is just an in progress draft that needs to be edited to something we want to ship.